src/index.html

```diff
@@ -1,13 +1,47 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page">
+    <section class="product-card" data-qa="card">
+      <img 
+        class="product-image" 
+        src="images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27 &apos; Retina 5K Monoblock"
+      >
+      <h1 class="product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="product-code">Product code: 195434</p>
+      <div class="client-review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="client-review__raiting">Reviews: 5</p>
+      </div>
+      <div class="price-section">
+        <p class="price-section__tag">Price:</p>
+        <p class="price-section__value">$2,199</p>
+      </div>
+      <form 
+        action="/" 
+        method="post" 
+        class="button-container"
+      >
+        <a href="/" data-qa="hover">
+          <button class="buy" type="submit">buy</button>
+        </a>
+      </form>
```

Comment:
No need for a form here.
Also, never wrap link into button. Use just `a` here.

Reply:
Fixed